src/containers/LocationPage/getIncidents.js

```diff
@@ -0,0 +1,115 @@
+
+/* NOTE: This file is unused.
+ *       its purpose is so that whoever trys to integrate the API
```

Comment:
"API" here might be too vague. Let's specify that it is the API between incidents stored on Google Cloud Storage in the file hierarchy and the Vis Client.